### DISCUSSION TITLE
So2sat processing steps

### DISCUSSION THREAD

**Discussion by AbdelghaniKABOT** [torchgeo/torchgeo#1250] [Q&A]
I trained a model on the So2sat dataset using TorchGeo and achieved excellent results on the testing split. However, when I tested it on new input data, the results were not as good. I think this may be due to differences in data processing. 
I only performed normalization using the following function:

```
def normalize(array):
    """Normalizes numpy arrays into scale 0.0 - 1.0"""
    array_min, array_max = array.min(), array.max()
    return ((array - array_min)/(array_max - array_min))
```

Can you help me with this issue?"

**Comment by adamjstewart** [comment]
You should always use the same normalization as the pre-trained model. See [here](https://github.com/microsoft/torchgeo/blob/v0.4.1/torchgeo/datamodules/so2sat.py) for the mean and std dev used to normalize images during training.
